### ISSUE TITLE
UEFI boot failure

### ISSUE THREAD

**Issue by seven62** [rocknsm/rock#293]
tested with different hardware and install drives:

UEFI option is available, and when selected the initial options screen displays showing:

   Install CentOS 7
   Test this media & install...
   Troubleshooting -->

after selection, the following error:

```
error: file `/images/pxeboot/vmlinuz` not found.
error you need to load the kernel first .
```

**Comment by jeffgeiger** [comment]
Pending testing with different/better USB drives.

**Comment by dcode** [comment]
Closed with rocknsm/rock-createiso#19
